src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSelector.js

```diff
@@ -183,5 +186,8 @@ export default compose(
         betas: {
             key: ONYXKEYS.BETAS,
         },
+        requestType: {
+            key: `${ONYXKEYS.SELECTED_TAB}_${CONST.TAB.RECEIPT_TAB_ID}`,
+        },
```

Comment:
NAB: I would call this selectedTab for clarity, especially because we compare it to a tab constant above.